playground/nextjs-app-router/components/demo/Swap.tsx

```diff
@@ -103,45 +103,108 @@ function SwapComponent() {
         </div>
       ) : null}
 
-      <Swap
-        className="w-full border sm:w-[500px]"
-        onStatus={handleOnStatus}
-        onSuccess={handleOnSuccess}
-        onError={handleOnError}
-        config={{
-          maxSlippage: defaultMaxSlippage || FALLBACK_DEFAULT_MAX_SLIPPAGE,
-        }}
-        isSponsored={isSponsored}
-      >
-        <SwapSettings>
-          <SwapSettingsSlippageTitle>Max. slippage</SwapSettingsSlippageTitle>
-          <SwapSettingsSlippageDescription>
-            Your swap will revert if the prices change by more than the selected
-            percentage.
-          </SwapSettingsSlippageDescription>
-          <SwapSettingsSlippageInput />
-        </SwapSettings>
-        <SwapAmountInput
-          label="Sell"
-          swappableTokens={swappableTokens}
-          token={ethToken}
-          type="from"
+      <div className="relative flex flex-col gap-6">
+        <h2>Swap Default</h2>
```

Comment:
since you're adding a default here, should we remove `SwapDefault` from the dropdown? 

<img width="252" alt="image" src="https://github.com/user-attachments/assets/1fc45792-251a-4d55-93c7-472b2abfd97b" />